src/main/java/com/example/spring/dto/book/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.spring.dto.book;
+
+import com.example.spring.validate.annotations.CoverImage;
+import com.example.spring.validate.annotations.Isbn;
+import jakarta.validation.constraints.Min;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class UpdateBookRequestDto {

```

Comment:
You can use CreateBookRequestDto to update book, the fields are the same

Reply:
Yeah, but i need all fields not null to create a book, but for update its not optimal to fill all fields in json. That's why i decided to create another DTO for updating book.